### ISSUE TITLE
Generating `Builder` fails on multiplatform

### ISSUE THREAD

**Issue by slelyuk** [square/wire#2416]
We're migrating our libraries to KMP and are widely using `Builder` from generated protos. In order to keep libraries backward compatible we've tried to use `javaInterop` flag:
```
wire {
    kotlin {
        javaInterop = true
    }
}
```
Now it generates `Builder`, but fails on every non-jvm target with error:

```
Unresolved reference: addUnknownFields
'build' overrides nothing
Unresolved reference: buildUnknownFields
```

Exactly the same error when applying `buildersOnly = true`.

Is this expected behavior? If so, is there any way to generate `Builder` for multiplatform project?

Environment:
Kotlin: 1.7.20
Wire: 4.4.3
Gradle: 7.4.2

**Comment by oldergod** [comment]
Builders are not currently supported for any platform but the JVM. We didn't expect other platforms to have any use of it actually.

**Comment by slelyuk** [comment]
@oldergod Got it. Thank you for clarification.

**Comment by oldergod** [comment]
Will be fixed with #2423
